source/Editor.js

```diff
@@ -1888,16 +1887,18 @@ var keyHandlers = {
 
         // Filter childnodes, that are textNodes.
         for (node=node.firstChild;node;node=node.nextSibling){
-            if (node.nodeType == 3) textNodes.push(node);
+            if (node.nodeType === 3 && node.textContet && node.textContent === " ") textNodes.push(node);
```

Comment:
Squire defines TEXT_NODE as a global within the closure (Squire is really one big closure). https://github.com/StileEducation/Squire/blob/master/source/Constants.js#L5. I'd like to float the idea of using this, instead of the Node.TEXT_NODE (I'm guessing that Neil did this for a reason).